src/main/java/com/chaos/invoicify/entity/InvoiceEntity.java

```diff
@@ -0,0 +1,47 @@
+package com.chaos.invoicify.entity;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
+@Getter
+@Setter
+@EqualsAndHashCode
+@NoArgsConstructor
+public class InvoiceEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    Long id;
+
+    @OneToMany(fetch = FetchType.LAZY,mappedBy = "invoice")
+    @JsonProperty(access = JsonProperty.Access.WRITE_ONLY)
+    List<ItemEntity> items;
+
+    String invoiceName;
+    String companyName;
```

Comment:
Company name should be replaced with real CompanyEntity type

Reply:
Done